artifacts/config_files/sfall-mods.ini

```diff
@@ -15,13 +21,13 @@ Containers=1
 ; 16 - bright yellow
 ; 32 - dark yellow
 ; 64 - purple
-OutlineColor=32
+OutlineColor=16
```

Comment:
Well, because it's the default color in previous sfall (other than configurable purple for containers), and it's the same color used for outlines if you mouse over items on the ground.